docs/src/manuals/meta-specification.md

```diff
@@ -1,52 +1,58 @@
 # [Meta Specification](@id user-guide-meta-specification)
 
-Some nodes in the `ReactiveMP.jl` inference engine accept optional meta structure that may be used to change or customise the inference procedure or the way node computes outbound messages. As an example `GCV` node accepts the approximation method that will be used to approximate non-conjugate relationships between variables in this node. `RxInfer.jl` exports `@meta` macro to specify node-specific meta and contextual information.
+Some nodes in the `RxInfer.jl` inference engine require a meta structure that may be used to either provide additional information to the nodes or customise the inference procedure or the way the nodes compute outbound messages. For instance, the `AR` node, modeling the Auto-Regressive process, necessitates knowledge of the order of the AR process, or the `GCV` node ([Gaussian Controlled Variance](https://ieeexplore.ieee.org/document/9173980)) needs an approximation method to handle non-conjugate relationships between variables in this node. To facilitate these requirements, `RxInfer.jl` exports `@meta` macro to specify node-specific meta and contextual information.
 
 ```@example manual_meta
 using RxInfer
 ```
 
 ## General syntax 
 
-`@meta` macro accepts either regular Julia function or a single `begin ... end` block. For example both are valid:
+`@meta` macro accepts either regular Julia `function` or a single `begin ... end` block:
 
 ```julia
-
-@meta function create_meta(arg1, arg2)
-    ...
+@meta function MyModelMeta(arg1, arg2)
+    Node(y,x) -> MetaObject(arg1, arg2)
 end
 
-mymeta = @meta begin 
-    ...
+my_meta = @meta begin 
+    Node(y,x) -> MetaObject(arg1, arg2)
 end
 ```
 
-In the first case it returns a function that returns meta upon calling, e.g. 
+In the first case it returns a function that returns meta upon calling. For example, the meta for the `AR` node of order $5$ can be specified as follows:
 
 ```julia
-@meta function create_meta(flag)
-    ...
+@meta function ARmodel_meta(num_order)
+    AR() -> ARMeta(Univariate, num_order, ARsafe())
 end
 
-mymeta = create_meta(true)
+my_meta = ARmodel_meta(5)
 ```
  
-and in the second case it returns constraints directly.
+In the second case it returns constraints directly. For example, the same meta for the `AR` node can also be defined as follows:
```

Comment:
Perhaps the AR example isn't needed in General syntax specification, but it would be helpful to see a model within which you need meta specification for the `Node(y, x)`